pkg/testutil/tempfile/tempfile.go

```diff
@@ -34,3 +39,40 @@ func (f *TempFile) Name() string {
 func (f *TempFile) Remove() {
 	os.Remove(f.Name())
 }
+
+var globalTestID string
```

Comment:
@AkihiroSuda yeah I just moved the code. I was wondering the same thing :sweat_smile: